EmbeddedForm/public/index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <title>Javascript USWDS Embedded Form</title>
+
+        <!-- Latest is 4.12.0 -->
+        <!-- Used on https://apps.form.io/uswds/ is 4.10.4 -->
+        <link rel="stylesheet" href="https://unpkg.com/formiojs@4.12.0/dist/formio.full.min.css" />
+        <script src="https://unpkg.com/formiojs@4.12.0/dist/formio.full.min.js"></script>
+        <!-- <link rel="stylesheet" href="https://unpkg.com/formiojs@4.10.4/dist/formio.full.min.css" />
+        <script src="https://unpkg.com/formiojs@4.10.4/dist/formio.full.min.js"></script> -->
+
+        <!-- Latest is 2.9.0 -->
+        <!-- Used on https://apps.form.io/uswds/ is 2.6.0 -->
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/uswds/2.9.0/css/uswds.min.css" />
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/uswds/2.9.0/js/uswds.min.js"></script>
```

Comment:
This looks redundant. You already added uswds.js and uswds.min.css below.

Reply:
There are two different USWDS items that need to be added.  The US Government supplied uswds (the one with version 2.9.0), and the form.io one - the one with version 1.8.2-rc.12.  I was definitely confused by that too.